app/static/src/app/store/run/state.ts

```diff
@@ -15,6 +16,39 @@ export interface ParameterSet {
     hidden: boolean
 }
 
+export enum AdvancedOptions {
+    "tol" = "Tolerance",
+    "maxSteps" = "Max Steps",
+    "stepSizeMax" = "Max Step Size",
+    "stepSizeMin" = "Min Step Size",
+    "tcrit" = "Critical Time"
```

Comment:
```suggestion
    "maxSteps" = "Max steps",
    "stepSizeMax" = "Max step size",
    "stepSizeMin" = "Min step size",
    "tcrit" = "Critical time"
```

Sentence case here looks more natural, and is what we have elsewhere within a panel (like "Log scale y axis").

How hard would it be (perhaps in another ticket) to get context sensitive help on these so that we can write a bit more about what these do?

Reply:
Yep, can change the case, what do you mean by context specific help, I was assuming they mean the same thing no matter what tab they are on right? Do you just mean tooltips or something on a help circle that show on hover? I don't think that would be super difficult to do! Can create a ticket for it once I confirm that I understand the intent

Reply:
just tooltips, using the existing approach from mrc-3694

Reply:
done: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4479/Context-help-for-advanced-settings